wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -261,6 +274,23 @@ def id(self) -> str:
         """Get the name of the custom object."""
         return self.name
 
+    def get_logs(self) -> Optional[str]:
+        """Get logs for custom object."""
+        # TODO: test more carefully once we release multi-node support
+        logs = {}
```

Comment:
```suggestion
        logs: Dict[str, Optional[str]] = {}
```

^^ or something like that, just nice to get typing help here.